evaluator/src/main/kotlin/RuleSet.kt

```diff
@@ -27,6 +27,7 @@ import com.here.ort.model.PackageReference
 import com.here.ort.model.Project
 import com.here.ort.model.RuleViolation
 import com.here.ort.model.Scope
+import com.here.ort.model.utils.collectLicenseFindings
```

Comment:
I'll approve once the commit message typo is fixed.

Reply:
Done.

Reply:
@sschuberth in addition to fixing that type I needed to add also one import to the reporter.

I now wonder whether importing the whole `utils` package is fine for now or whether it is worth introducing a separate package. Any opinion?   (I wanted at least to highlight that changed import so that you are aware)

Reply:
As it's only imported in scripts via a wildcard-import and scripting is all about convenience, I'm fine with not introducing a separate package for now.